src/OrchardCore.Themes/TheAgencyTheme/Views/Shared/Search.liquid

```diff
@@ -9,6 +9,12 @@
 
 {{ Model.SearchForm | shape_render }}
 
+{% unless Model.SearchResults %}
+  {% if Model.Terms %}
+    {% helper "span", validation_for: "Terms", class: "alert alert-danger" %}
+  {% endif %}
+{% endunless %}
+
```

Comment:
This liquid block and the similar one in the Blog theme template are needed since I can't find a Liquid equivalent of `ModelState.IsValid`. If there is no search results shape but there are search terms, there must be a validation error.

Reply:
@Skrypt can you review? I am happy with the update from @thomasebsmith 

Sorry @thomasebsmith I think I already hassled you on the last pr about the `ToList()` :)

Reply:
I need to look at this to approve PR first. It's probably working but maybe there's a better way to do this.